src/Jobs_Templete/deployment.yaml.template

```diff
@@ -37,7 +37,17 @@ spec:
       - name: main
         image: {{ job["image"] }}
         imagePullPolicy: Always
-        command: ["sh", "-c", "printenv DLTS_LAUNCH_CMD > /pod/job_command.sh ; bash -x /pod/job_command.sh"]
+        command: ["sh", "-c",
+        "
+        printenv DLTS_LAUNCH_CMD > /job_command.sh ;
```

Comment:
Just to be cautious, is this tested in dev cluster?

Reply:
yes